src/parser/Statement.ts

```diff
@@ -53,7 +54,8 @@ export class EmptyStatement extends Statement {
  */
 export class Body extends Statement implements TypedefProvider {
     constructor(
-        public statements: Statement[] = []
+        public statements: Statement[] = [],
+        public parser?: Parser
```

Comment:
This is a hack part 1
Basically, I just want to be able to get tokens from an expression statement, so I can look for comments
Is there a better way to do this?